### PR TITLE
Grid fix to enforce new row every 3 devices

### LINKED ISSUE #194: Missing Section in Devices Grid
On the devices page, there are empty sections in the grid. I don't know if this is die to missing devices, or just layout issues.

firefox_screenshot_2015-11-18t11-28-25 507z

I see with all devices, and with filtered (screenshot).

### PR DESCRIPTION
**Pull request by comzeradd**
- Fixes #194

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: feedthefox/devices/templates/jinja2/devices.html [hunk 1]
```diff
@@ -16,24 +16,31 @@
...
+              </div>
+              <div>
+                <p>{{ device }}</p>
+              </div>
+            </a>
+          </div>
+        {% if loop.index is divisibleby 3 %}
+          </div>
+           {% if not loop.last %}
+             <div class="row">
```
Comment (akatsoulas) [inline feedthefox/devices/templates/jinja2/devices.html]: It seems that there is a `</div>` missing here.

File: feedthefox/devices/templates/jinja2/devices.html [hunk 2]
```diff
@@ -16,24 +16,31 @@
...
-          </a>
-        </div>
+                {% endif %}
+              </div>
+              <div>
+                <p>{{ device }}</p>
+              </div>
+            </a>
+          </div>
+        {% if loop.index is divisibleby 3 %}
```
Comment (akatsoulas) [inline feedthefox/devices/templates/jinja2/devices.html]: Maybe you could use the `batch` filter here and group devices by 3.

File: feedthefox/devices/templates/jinja2/devices.html [hunk 3]
```diff
@@ -16,24 +16,31 @@
...
+        {% for device in device_filter %}
+          <div class="col-sm-4">
+            <a href="{{ url('device', manufacturer=device.manufacturer, model=device.model) }}"
+               class="device-list">
+              <div>
+                {% if device.image %}
+                  <img src="{{ device.image.url }}"
+                       class="img-responsive device-img" alt="{{ device.model }}">
+                {% else %}
+                <img src="{{ static(settings.DEFAULT_DEVICE_IMG) }}"
```
Comment (akatsoulas) [inline feedthefox/devices/templates/jinja2/devices.html]: Indent please.

**Comment by comzeradd** [comment]
@akatsoulas PR updated. Thanks for the `batch` suggestion. It makes the far more readable.

**Comment by akatsoulas** [comment]
r+ :guitar:

**Pull request merged**